app/uk/gov/hmrc/gform/eval/ExpressionResult.scala

```diff
@@ -135,6 +147,7 @@ sealed trait ExpressionResult extends Product with Serializable {
     case t: DateResult    => false
     case t: PeriodResult  => er.ifPeriodResult(t.value.toTotalMonths > _.toTotalMonths)
     case t: AddressResult => false
+    case t: ListResult    => er.ifNumberResult(n => t.list.exists(_.ifNumberResult(_ > n)))
```

Comment:
This is missing `>` operation on String, Period and Option results.
Again better implementation seems to be
```
t.list.exists(_.>(er))
```

Reply:
Hmm, I didnt want it for other values as > is mainly used for numbers only. Not sure why one would use > for string or option, but Period will be useful. Your impl seems to handle all